metis-authentication/metis-authentication-rest/docker-compose.yml

```diff
@@ -11,18 +12,17 @@ services:
       - '5432:5432'
   metis-authentication-local:
     image: europeana/metis-authentication:develop
-    container_name: metis-authentication
+    container_name: metis-authentication-local
     build:
       context: ./
       dockerfile: Dockerfile
     ports:
       - '8080:8080'
-    #uncomment this if an .env file is to be used instead of a application.properties.
-    #    env_file:
-    #      - /data/metis-properties/test/metis-authentication-rest/.metis-authentication-test.env
+    environment:
+      HIBERNATE_CONNECTION_URL: jdbc:postgresql://metis-authentication-postgres:5432/metis-authentication_local
     volumes:
-      - /data/certificates/ibm-ca-truststore.jks:/data/certificates/ibm-ca-truststore.jks
-      - /data/metis-properties/test/metis-authentication-rest/application-test.properties:/application.properties
-      - /data/logging/log4j2.xml:/log4j2.xml
```

Comment:
No I don't think it will often change.
But it might change in the coming migrations until we have something more stable.
So far there has been a lot of trial and error from my side.
Environment variables are a good idea too, if they work through intellij, since I had issues with it tbh and that's why they are actually setup here.
So for now I'd say let's still to this and we'll see how it evolved in the next half a year or so.
What do you think?

Reply:
is ok, it is an refining state so is ok to stay like this.